.github/workflows/test_fastlane_build.yaml

```diff
@@ -45,6 +35,17 @@ jobs:
       #   run: bundle exec fastlane add_linear_comment from_tag:dev/0.22.0/32 to_tag:dev/0.22.0/33 comment:"Test comment \`0.23\`"
       #   env:
       #    LINEAR_TOKEN: ${{ secrets.LINEAR_TOKEN }}
+         
+      # Test GITHUB_TOKEN permissions
```

Comment:
Tested this step on
https://github.com/anyproto/anytype-swift/actions/runs/7927754415